src/styles/blocks/stars.css

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(../../images/star.svg);
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
```

Comment:
Be consistent with your margins. It's recommended to add only top or bottom margins, but not both, to avoid potential margin collapse issues. If the margin is intended to space the stars horizontally, it's fine, but ensure that vertical margins are not mixed in elsewhere without good reason.